index.js

```diff
@@ -1,9 +1,15 @@
 const Condition = require('./src/conditions/Condition')
 const Workflow = require('./src/workflow')
 const Event = require('./src/eventSourcing/Event')
+const EventSourcing = require('./src/eventSourcing/EventSourcing')
+const EventGenerator = require('./src/test-helpers/EventGenerator')
+const WorkflowGenerator = require('./src/test-helpers/WorkflowGenerator')
```

Comment:
decided to expose the test helpers as source. Will come handy in our other repos.